script/deploy-dev.mk

```diff
@@ -40,10 +44,16 @@ deploy-dev: deploy-dependencies
 	@echo "  kubeapps-user-ldap@example.org:password"
 	@echo "to authenticate with the corresponding permissions."
 
+reset-dev-kubeapps:
+	helm --kubeconfig=${CLUSTER_CONFIG} -n kubeapps delete kubeapps
+	kubectl -kubeconfig=${CLUSTER_CONFIG} delete jobs -n kubeapps --all
+	kubectl delete namespace --wait kubeapps
```

Comment:
Is there a reason the first two lines are needed? I normally just delete the namespace (which delete the helm release as it's a secret in the ns, and deletes all the jobs)?

Reply:
Well, there is no strong reason besides my impatience... when deleting the ns, I have to keep waiting without any information about what's happening. However, if I manually delete the helm release, then the jobs and finally, everything else, I receive more information and feel happier to see each step succeeding.

But, as for the makefile target, if you think it's better to just have the ns deletion since it's clearer, go ahead, I don't mind.

Reply:
I don't care either way - I was just interested to know. Feedback on the process is good, definitely.